india_compliance/gst_india/overrides/sales_invoice.py

```diff
@@ -106,10 +106,10 @@ def validate_fields_and_set_status_for_e_invoice(doc, gst_settings=None):
     )
 
     # Mandatory for e-Invoice before save
-    _validate_hsn_codes(
```

Comment:
e-Invoice is applicable above 5cr.
above 5cr, it's mandatory to use 6 digits. still users may voluntarily apply e-Invoice, for which we can allow 4 digits here.